data-transformers/transformations/dataTransformations.py

```diff
@@ -1,3 +1,3 @@
 class Transformations():
-    def secondsToMilliseconds(s):
+    def secondsToMilliseconds(self, s):
```

Comment:
I'm using the functions in Transformations by doing an eval to call the right one based on the configuration file for the transformation to apply.